### PR TITLE
Scenario 32 changes

### PR DESCRIPTION
**Pull request by EnSnack**
Added the aforementioned retaliate from #113 though it doesn't calculate it on the fly.

### CODE REVIEW

**Review by Lurkars — CHANGES_REQUESTED** [review]

File: data/fh/sections.json [hunk 1]
```diff
@@ -427,8 +427,21 @@
...
       {
-        "name": "Raven Nest a",
-        "health": "(L+2)xCx3"
+        "name": "Raven Nest",
+        "health": "(L+2)xCx3",
+        "marker": "a",
+        "actions": [
+          {
+            "type": "retaliate",
+            "value": "(L+1)/3 round up",
```
[the remark this answers is not in the document]
Comment (EnSnack) [reply data/fh/sections.json:436]: chrome_2023-02-06_19-52-33

I tried this originally before just doing this but it seems to take it literally as a string, I'm guessing it doesn't get sifted through the entity replacer.

[the remark this answers is not in the document]
Comment (Lurkars) [reply data/fh/sections.json:436]: Okay, please change anyway and commit. I investigate the issue why it's not applied and will ship a fix before making a release with this PR!

**Review by Lurkars — CHANGES_REQUESTED** [review]

File: data/fh/sections.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Lurkars) [reply data/fh/sections.json]: (Info this is direct related to the changes for #112)

**Review by Lurkars — APPROVED** [review]

**Comment by Lurkars** [comment]
I merged this `frozen-corpse-scenario-32.json` for now. But I think, I will replace this in feature to actual implement this as special rules! For now it's better than not having it. I am too busy with fixing/investigating issues by now, so I don't find time for the data/rules currently.

**Pull request merged**